integration-tests/qute/src/test/java/io/quarkus/it/qute/QuteTestCase.java

```diff
@@ -28,6 +28,18 @@ public void testTemplates() throws InterruptedException {
                 .body(containsString("Hello Ciri!"));
         RestAssured.when().get("/beer").then().body(containsString("Beer Pilsner, completed: true, done: true"));
         RestAssured.when().get("/defaultmethod").then().body(containsString("Hello MK"));
+        RestAssured
+                .given()
+                .contentType("application/json")
+                .body("""
+                        {
+                          "name": "foo"
+                        }
+                        """)
+                .when().post("/json")
+                .then()
+                .statusCode(200)
+                .body(containsString("foo"));
```

Comment:
Could we add the test in something more core than the Qute ITs. Reasoning is that if at some point we drop Qute, we will lose these tests which are related to basic core features.

(Yeah, sorry for being annoying!)

Reply:
I looked for another place and didn't find a good candidate. 

Agreed it's not a good place, but I have no better candidates in mind